tests/jbuild

```diff
@@ -0,0 +1,22 @@
+
+(executable
+  ((name test_toml)
+   (libraries (oUnit toml threads bisect))))
+
+(alias
+  ((name runtest)
+   (deps (test_toml.exe ./example.toml ./example4.toml))
+   (action (run ./test_toml.exe))))
+   
+(alias
+  ((name runtest)
+   (deps (test_toml.exe))
+   (action
+     (progn
+       (setenv BISECT_FILE coverage
+         (run ./test_toml.exe))
+     (run sh -c "bisect-report -I .*.eobjs -verbose -html report coverage*.out")
+     ))))
+;(executable
+;  ((name parser_with_coverage)
+;   (libraries (ounit toml bisect))))
```

Comment:
is this code dead ?

In this case: why keep it ?

Reply:
I'm not sure what it's supposed to do/output. I haven't used bisect in a while…

Reply:
if it's dead code, delete it. It's simpler